app/format_json.py

```diff
@@ -115,12 +115,20 @@ def main():
         reader = csv.reader(csvfile)
         for row in reader:
             keys = row[0].split('.')
-            merge(data, format_dict(keys, row[1]))
+            merge(data, format_dict(keys, row[1].strip(" ")))
```

Comment:
`str.strip` defaults to removing whitespace: https://docs.python.org/2.7/library/stdtypes.html#str.strip, so passing `" "` as an argument may be unnecessary.